src/main.js

```diff
@@ -24,9 +24,9 @@ const checkValidationOfValues = envSpecString => {
   ];
   const alphanumericThatDoesNotStartWithDigit = /^[A-Z_][0-9A-Z_]*$/;
   const genericForCheckingRestrChoicesSyntax = /^\[(.*)\]$/;
+  const allowedTypes = validTypes.join(",");
```

Comment:
Since we are are referencing the allowed env spec types in text we want to put each reference (type) in quotes. This could work:

```js
const allowedTypes = validTypes.map(validType => `"${validType}"`).join(",");
```